cmd/generate-makefile/funcmaps.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2023 Chaos Mesh Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import "strings"
+
+var defaultFuncMap = map[string]interface{}{"StringsJoin": StringsJoin}
+
+func StringsJoin(s []string, sep string) string {
+	return strings.Join(s, sep)
```

Comment:
It is better to check for empty strings.

Reply:
`strings.Join` already takes care about that :P

https://github.com/golang/go/blob/e8fe3b77574d9afadab664104a669d8e9f0f4303/src/strings/strings.go#L432-L464

Reply:
I would mark this comment as resovled